app/models/user.rb

```diff
@@ -55,14 +59,16 @@ def gravatar_url
     "https://www.gravatar.com/avatar/#{md5}"
   end
 
-  Role.all.each do |role|
-    define_method "is_#{role.name}?" do
-      role_id >= role.id
-    end
+  def is_admin_for?(project)
+    user_project_roles.find_by(project: project).try(:is_project_admin?)
+  end
+
+  def is_deployer_for?(project)
+    user_project_roles.find_by(project: project).try(:is_project_deployer?)
   end
 
-  def role
-    Role.find(role_id)
+  def project_role_for(project)
+    user_project_roles.find_by(project: project)
```

Comment:
You're talking about the method:  `project_role_for` ? Is being used in the `UsersHelper`.